academic-services/graph/samples-azure-databricks-hindex.md

```diff
@@ -0,0 +1,337 @@
+
+   ```
+   +--------+--------------------+----------------------+----------+
+   |AuthorId|         DisplayName|LastKnownAffiliationId|PaperCount|
+   +--------+--------------------+----------------------+----------+
+   |     584|Gözde Özdikmenli-...|              79946792|         2|
+   |     859|          Gy. Tolmár|                  null|         2|
+   |     978|      Ximena Faúndez|             162148367|        18|
+   ...
+   ``` 
+
+1. Get **(Author, Paper) pairs**. Paste the following code in a new cell.
+
+   ```python
+   PaperAuthorAffiliationsPath = 'mag/PaperAuthorAffiliations.txt'
+   PaperAuthorAffiliationsFields = ['PaperId', 'AuthorId', 'AffiliationId', 'AuthorSequenceNumber', 'OriginalAffiliation']
+   PaperAuthorAffiliations = spark.read.format('csv').options(header='false', inferSchema='true', delimiter='\t').load(('%s/%s' % (MagDir, PaperAuthorAffiliationsPath))).toDF(*PaperAuthorAffiliationsFields)
+
+   AuthorPaper = PaperAuthorAffiliations.select(PaperAuthorAffiliations.AuthorId, PaperAuthorAffiliations.PaperId).distinct()
+   AuthorPaper.show(10)
+   AuthorPaper.createOrReplaceTempView('AuthorPaper')
+   ```
+
+   Press the **SHIFT + ENTER** keys to run the code in this block. You see an output similar to the following snippet:
+
+   ```
+   +----------+--------+
+   |  AuthorId| PaperId|
+   +----------+--------+
+   |2121966975|94980387|
+   |2502082315|94984326|
+   |2713129682|94984597|
+   ...
+   ...
+   ``` 
+
+1. Get **Papers**. Paste the following code in a new cell.
+
+   ```python
+   PapersPath = 'mag/Papers.txt'
+   PapersFields = ['PaperId', 'Rank', 'Doi', 'DocType', 'PaperTitle', 'OriginalTitle', 'BookTitle', 'Year', 'Date', \
+                   'Publisher', 'JournalId', 'ConferenceSeriesId', 'ConferenceInstanceId', 'Volume', 'Issue', 'FirstPage', \
+                   'LastPage', 'ReferenceCount', 'CitationCount', 'EstimatedCitation', 'OriginalVenue', 'CreatedDate']
+   Papers = spark.read.format('csv').options(header='false', inferSchema='true', delimiter='\t').load(('%s/%s' % (MagDir, PapersPath))).toDF(*PapersFields)
+
+   PaperCitation = Papers.select(Papers.PaperId, Papers.EstimatedCitation).where(Papers.EstimatedCitation > 0)
+   PaperCitation.show(10)
+   PaperCitation.createOrReplaceTempView('PaperCitation')
+   ```
+
+   Press the **SHIFT + ENTER** keys to run the code in this block. You see an output similar to the following snippet:
+
+   ```
+   +----------+-----------------+
+   |   PaperId|EstimatedCitation|
+   +----------+-----------------+
+   |2088151486|               61|
+   |2864100843|                1|
+   |2260674751|                5|
+   ...
+   ...
+   ``` 
+
+   You have now extracted MAG data from Azure Storage into Azure Databricks and created temporary views to use later.
+
+## Compute Author H-Index
+
+In this section, you compute h-index for all authors.
```

Comment:
Nit: It might be worth having an external link to the definition of h-index (https://en.wikipedia.org/wiki/H-index)

Reply:
Thanks for the suggestion. Added.